dask_geomodeling/geometry/aggregate.py

```diff
@@ -110,13 +110,164 @@ def bucketize(bboxes):
     ]
 
 
+def aggregate_polygons(
```

Comment:
This function is a *mostly* unchanged copy from `AggregateRaster.process`. I marked the lines that I changed.